notifications/client_test.go

```diff
@@ -16,7 +16,7 @@ func TestRequestSuccess(t *testing.T) {
 	}))
 	defer server.Close()
 
-	c := CreateClient()
+	c := CreateClient("")
```

Comment:
thanks for the pr :)

just nit picking here but I think passing nil and using *string is more idiomatic. do you mind changing it?

Reply:
changed